sigstore-java/src/main/java/dev/sigstore/tuf/Updater.java

```diff
@@ -78,6 +83,7 @@ public class Updater {
     this.localStore = localStore;
     this.metaFetcher = metaFetcher;
     this.targetFetcher = targetFetcher;
+    this.trustedMeta = TrustedMeta.newTrustedMeta(localStore);
```

Comment:
since it's a read-through cache, could it be simpler to just invalidate it on update or something rather than having the updater have to know about it?

Reply:
Oh like if the root updated?

Reply:
Yeah probably